practice/bio/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html lang= "en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Bio</title>
+  </head>
+
+  <body>
+    <header>
+      <h1>John Cena Bio</h1>
+    </header>
+    <nav>
+    </nav>
+    <section>
+      <p>
+        John Felix Anthony Cena Jr. (born April 23, 1977) is an American professional wrestler, rapper, actor, and reality television show host signed to WWE on the SmackDown brand.
+      </p>
+      <img src="johncena.jpg">
+    </section>
+    <section>
+      <h2>Places lived in</h2>
+      <p>
+        John Cena was born in West Newbury, Massachusetts. He currently resides in Tampa, Florida.
+      </p>
+    </section>
+    <section>
+      <blockquote>
+        <em>I don't doll myself up for TV because I want people to accept me for who I am.</em>
```

Comment:
This `em` tag is for style. I know it doesn't look like what you want at this point, because there's no CSS, but in the future the block quote should be styled directly.